airflow/providers/google/cloud/hooks/kubernetes_engine.py

```diff
@@ -352,6 +353,7 @@ def __init__(
         self,
         cluster_url: str,
         ssl_ca_cert: str,
+        disable_tcp_keepalive: bool | None = None,
```

Comment:
The default behavior should be  tcp_keepalive disabled.
```suggestion
        disable_tcp_keepalive: bool | None = True,
```

Reply:
I don't think so, this behavior repeats the original behavior which we have for `KubernetesHook`
https://github.com/apache/airflow/blob/main/airflow/providers/cncf/kubernetes/hooks/kubernetes.py#L130

Reply:
Yah, you are right the default behavior is same.

Reply:
I have hard time understanding why None is allowed value (since it seems that just having default False and boolean should be fine, I am also not a big fan  of disable_* flags in general and having to have strange conditions(`is not True` is a condition that it's difficult to wrap your head around) - but yeah in this case consistency trumps logic.